### DISCUSSION TITLE
error: is not assignable to type FieldError when use subobjects with yup

### DISCUSSION THREAD

**Discussion by leonidasroberto** [react-hook-form/react-hook-form#9711] [Ideas]
Error when use subobjects to yup error: 
Type error: Type 'Merge<FieldError, Merge<FieldError, FieldErrorsImpl<{ some object here }>>[]>' is not assignable to type 'FieldError'.

Solution/Sugestion:
on errors.ts change line 14 to:   type?: LiteralUnion<keyof RegisterOptions, string>;

**Comment by leapful** [comment]
@leonidasroberto 

Please follow the template for asking question https://github.com/react-hook-form/react-hook-form/issues/2848 including provide a codesandbox url to re-produce the issue.
